apps/iglidur-designer/src/app/pages/model/model.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, OnDestroy, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
```

Comment:
This is not what the feature is supposed to be doing

Reply:
I had already implemented a model viewer so it formed part of the files for the PR.
This was before you told me to focus on the other features.

Reply:
Do not mix features in one PR. You can always commit files that are solely about what you are doing